Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-filter/ContestFilter.tsx

```diff
@@ -13,15 +13,19 @@ import styles from './ContestFilter.module.scss';
 interface IContestFilterProps {
     values: IFilter[];
     type: FilterType;
-    onSelect: (filterId: number) => void;
+    onSelect: (filterId: number, filterType: FilterType) => void;
     maxDisplayCount: number;
+    statusFilterId: number;
+    strategyFilterId: number;
 }
 
 const ContestFilter = ({
     values,
     type,
     onSelect,
     maxDisplayCount,
+    statusFilterId,
+    strategyFilterId,
```

Comment:
This is available in `use-contest`. There is a `state` that holds the currently selected filters